waypoints_trajectory: 
  positions:
  - [0.0, 0.0, 0.0, 0.0]
  - [1.0, 1.0, 1.0, 0.5]
  - [2.0, -2.0, 1.0, 0.5]
  - [1.0, 1.0, 4.0, 0.2]
  - [0.0, 0.0, 0.0, 0.0]
  time_from_start:
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  
max_velocity: 3.0
delta_time: 0.001
goal_tolerance: 0.0
path_tolerance: 0.0001